### ISSUE TITLE
Contact owner of wireles redstone

### ISSUE THREAD

**Issue by Greazi-Times** [licks92/WirelessRedstone#219]
Hi there I have a simple question to all of you I want to get in touch with the owner. If you have some info plzz contact me

**Comment by SpaceNinjaApe** [comment]
I don't have any contact info just wunt to bump this since i really think this should be continued maybe fork this? Is there any alternative?

**Comment by axelpey** [comment]
This is the owner. I realize it's been a long time since I've looked at this code, I'll take a look around and see if the open PR manages to make it work.

**Comment by SpaceNinjaApe** [comment]
I've been using your plugin a lot back in the day. I'am also an experienced programmer and glad to help if needed.

**Comment by Greazi-Times** [comment]
I love to help as well to bring back life in this project I have used this plugin for so long it still worked up to 1.16 but than it stopped working. 😁

**Comment by axelpey** [comment]
Yeah it was also one of my first projects, but I'd be happy to come back to it and see if we can revive it. Lost experience in Java though, even more with the bukkit framework so I would really benefit from your help @SpaceNinjaApe.

EDIT: I was also not receiving notifications before, but I just fixed that. Now ready to get back to it with your help.

**Comment by Greazi-Times** [comment]
> Yeah it was also one of my first projects, but I'd be happy to come back to it and see if we can revive it. Lost experience in Java though, even more with the bukkit framework so I would really benefit from your help @SpaceNinjaApe.
> 
> 
> 
> EDIT: I was also not receiving notifications before, but I just fixed that. Now ready to get back to it with your help.

I have been using the plugin a lot and have been coding for 2 years on Minecraft plugins take a look at TechsCode where I am a dev/assistant to help out with coding.

**Comment by axelpey** [comment]
> > Yeah it was also one of my first projects, but I'd be happy to come back to it and see if we can revive it. Lost experience in Java though, even more with the bukkit framework so I would really benefit from your help @SpaceNinjaApe.
> > EDIT: I was also not receiving notifications before, but I just fixed that. Now ready to get back to it with your help.
> 
> I have been using the plugin a lot and have been coding for 2 years on Minecraft plugins take a look at TechsCode where I am a dev/assistant to help out with coding.

Interesting project! What do you have in mind for the continuation of WirelessRedstone though? Transfer of ownership under TechsCode ?

**Comment by Greazi-Times** [comment]
> > > Yeah it was also one of my first projects, but I'd be happy to come back to it and see if we can revive it. Lost experience in Java though, even more with the bukkit framework so I would really benefit from your help @SpaceNinjaApe.
> 
> > > EDIT: I was also not receiving notifications before, but I just fixed that. Now ready to get back to it with your help.
> 
> > 
> 
> > I have been using the plugin a lot and have been coding for 2 years on Minecraft plugins take a look at TechsCode where I am a dev/assistant to help out with coding.
> 
> 
> 
> Interesting project! What do you have in mind for the continuation of WirelessRedstone though? Transfer of ownership under TechsCode ? 

No, just keep it a separate project.
We can however add GUI's to manage parts of the plugin. Like see all the signs and spots.
I would suggest keeping the old setup and fix all the bugs there are same as optimizing the whole plugin as the plugins that is now online works good up to 1.16.
A more modern view with GUI's would make the plugin look already a bit more up to date even it is just for now all the signs and being able to teleport to them.

**Comment by Greazi-Times** [comment]
What would you guys think of making a Discord group?
If you have discord at least. Talkes a little easier than here in my opinion.

**Comment by axelpey** [comment]
Alright I think just fixing what broke it is already a good first goal, because I will not be able to commit a lot of time to this, I'm very busy this fall. For the GUI aspect, I most definitely will not be able to take a part in that but I'll help contribute if you want, and make sure you're recognized as a big contributor to the project.

**Comment by axelpey** [comment]
And regarding discord, I'm down

**Comment by Greazi-Times** [comment]
If you add me to discord we can create a group, 'Greazi#9999'

**Comment by Greazi-Times** [comment]
If there is anyone that would like to proceed with this I would like to hear from you guys 😁

**Comment by axelpey** [comment]
Alright @Greazi-Times let me add you to Discord. It was a crazy month for me but I feel like I could spend some time on this now.

**Comment by rackgaming** [comment]
Are you guys bringing this bad boy back? I would absolutely love to use it in alternative to lots of redstone to reduce redstone, it sounds like an amazing plugin and I have used it but it doesn't sadly work with BigDoor golden power blocks :( And I am using paper 1.19.2 so surprisingly it works at all lol because of viaversion and viabackwards.

**Comment by whitemoon75** [comment]
An update to this would be great, i only use 3 mods on my server, this is one.

**Comment by axelpey** [comment]
@Bart-0110 we have an update made with the help of @FakeDeath94

Could you give me admin rights on Spigot so that we can publish it? We'll also take care of updating the code here properly from his fork

My profile: https://www.spigotmc.org/members/licks92.2079359/

**Comment by Greazi-Times** [comment]
> @Bart-0110 we have an update made with the help of @FakeDeath94
> 
> 
> 
> Could you give me admin rights on Spigot so that we can publish it? We'll also take care of updating the code here properly from his fork
> 
> 
> 
> My profile: https://www.spigotmc.org/members/licks92.2079359/

Just create a pull request to this repo and then he can update it him self.

**Comment by axelpey** [comment]
> > @Bart-0110 we have an update made with the help of @FakeDeath94
> > Could you give me admin rights on Spigot so that we can publish it? We'll also take care of updating the code here properly from his fork
> > My profile: https://www.spigotmc.org/members/licks92.2079359/
> 
> Just create a pull request to this repo and then he can update it him self.

I am owner of this repo so this won't be an issue, the issue is to publish to Spigot
